docs/user/customization.md

```diff
@@ -384,6 +384,70 @@ For example:
     worker-edab0895c59dba7a566f4b955d87d964                     3.11.0-744-g5b05d9d3-dirty   2.2.0             137m
     ```
 
+#### Nodes with Custom Kernel Arguments
+
+Custom kernel arguments can be applied to through manifests as an installer operation, and can also be applied as a [MachineConfig][machine-config] as a day 2 operation. The kernel arguments are applied upon boot and will be honored by the Machine-Config-Operator from then on.
+
+Example application of `nosmt` (disable hyperthreading):
+
+1. Run `manifests` target to create all the manifests.
+
+    ```console
+    $ mkdir nosmt-cluster
+
+    $ openshift-install --dir nosmt-cluster create manifests
+    ...
+
+    $ ls -l nosmt-cluster/openshift
+    ...
+    99_openshift-machineconfig_99-master-ssh.yaml
+    99_openshift-machineconfig_99-worker-ssh.yaml
+    ...
+    ```
+
+2. Edit the master/worker machineconfig to add the desired kernel argument (in list format):
```

Comment:
dropping new machineconfigs is more desirable here rather than changing the existing ssh ones.